### PR TITLE
[SOAR-747] - Delete email with slack remove join step

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Description

This PR removes the join step in the Delete O365 Emails with Slack workflow

## Workflow: 
workflow1
workflow2

### CODE REVIEW

File: workflows/Delete_O365_Emails_With_Slack/Delete_O365_Emails_With_Slack.icon [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
   "kom": {
-    "komandVersion": "1.48.2-0-g4eb3f",
+    "komandVersion": "1.48.4-0-g02625",
     "komFileVersion": "2.0.0",
-    "exportedAt": "2020-04-07T17:15:42.958451022Z",
+    "exportedAt": "2020-04-17T15:49:17.460631553Z",
     "workflowVersions": [
       {
-        "name": "Delete O365 Emails With Slack",
+        "name": "Delete Office 365 Emails with Slack",
```
Comment (mrinehart-r7) [inline workflows/Delete_O365_Emails_With_Slack/Delete_O365_Emails_With_Slack.icon:8]: You'll need to rename the directory and filename if you want to make this change.

File: workflows/Delete_O365_Emails_With_Slack/workflow.spec.yaml [hunk 2]
```diff
@@ -15,12 +15,10 @@ resources:
   source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Delete_O365_Emails_With_Slack
   license_url: https://github.com/rapid7/insightconnect-workflows/blob/master/LICENSE
   screenshots:
-  - name: screenshot1.png
```
Comment (mrinehart-r7) [inline workflows/Delete_O365_Emails_With_Slack/workflow.spec.yaml:18]: Make sure to remove the screenshot1/2/3.png files

File: workflows/Delete_O365_Emails_With_Slack/workflow.spec.yaml [hunk 3]
```diff
@@ -15,12 +15,10 @@ resources:
...
+  - name: workflow1.png
     title: Workflow 1
-  - name: screenshot2.png
+  - name: workflow2.png
     title: Workflow 2
-  - name: screenshot3.png
-    title: Workflow 3
```
Comment (mrinehart-r7) [inline workflows/Delete_O365_Emails_With_Slack/workflow.spec.yaml:23]: I would rename these titles so as not to confuse the user (they may think there are multiple workflows with this naming scheme). Something like "Workflow Part 1/2/3/etc" would be less confusing imo.

File: workflows/Delete_O365_Emails_With_Slack/workflow.spec.yaml (hunk 2 above)
Comment (jmcadams-r7) [reply workflows/Delete_O365_Emails_With_Slack/workflow.spec.yaml:18]: They're gone.

**Pull request merged**